### DISCUSSION TITLE
apt update fails with ppa2.novemberain.com: Mirror sync in progress?

### DISCUSSION THREAD

**Discussion by gulbinas** [rabbitmq/rabbitmq-server#9956] [Questions] [answered]
Installed rabbitmq-server via this script: https://www.rabbitmq.com/install-debian.html#apt-quick-start-cloudsmith

now I get error when i do `apt update`

E: Failed to fetch https://ppa2.novemberain.com/rabbitmq/rabbitmq-server/deb/ubuntu/dists/jammy/main/binary-amd64/Packages.gz  File has unexpected size (12744 != 12797). Mirror sync in progress? [IP: 172.67.140.63 443]
   Hashes of expected file:
    - Filesize:12797 [weak]
    - SHA256:1cf12271f63b0b1d0f2b23d4ab85fd4cfff442029c325c4590d4a0b12854e86c
    - SHA1:233692f6ec56335b847761bc5b4e2adbe2964fc4 [weak]
    - MD5Sum:d9e00d71e3044d93ff12445b494d4db3 [weak]
   Release file created at: Fri, 17 Nov 2023 08:18:46 +0000
E: Some index files failed to download. They have been ignored, or old ones used instead.

**Comment by michaelklishin** [comment]
There indeed was a sync in progress.

**Reply by michaelklishin** [reply, marked as answer]
Generally this does not last for more than a few minutes, for example, Cloudflare cache is set to 2 minutes.

[Specifying both repositories](https://rabbitmq.com/install-debian.html#apt-quick-start-cloudsmith) also helps because they are never in the process of syncing at the same time.
